pkg/operator/controller/clustercompliancereport.go

```diff
@@ -0,0 +1,88 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"github.com/aquasecurity/starboard/pkg/apis/aquasecurity/v1alpha1"
+	"github.com/aquasecurity/starboard/pkg/compliance"
+	"github.com/aquasecurity/starboard/pkg/operator/etc"
+	"github.com/aquasecurity/starboard/pkg/operator/predicate"
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+type ClusterComplianceReportReconciler struct {
+	logr.Logger
+	etc.Config
+	client.Client
+	compliance.Mgr
+}
+
+func (r *ClusterComplianceReportReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	installModePredicate, err := predicate.InstallModePredicate(r.Config)
+	if err != nil {
+		return err
+	}
+	err = ctrl.NewControllerManagedBy(mgr).
+		For(&v1alpha1.ClusterComplianceReport{}, builder.WithPredicates(
+			predicate.Not(predicate.IsBeingTerminated),
+			installModePredicate)).
```

Comment:
Why do we need a predicate that checks namespaces for a cluster-scoped resource? Actually, without commenting out these predicates and fixing RBAC permission to let starboard update the clustercompliancereports/status resource it doesn't work.